src/common/localization/en-us.ts

```diff
@@ -29,6 +29,12 @@ export const english: IAppStrings = {
             title: "Delete Project",
             confirmation: "Are you sure you want to delete project",
         },
+        importProject: {
+            title: "Import Project",
+            confirmation: "Are you sure you want to convert project",
```

Comment:
What was the purpose of separating these two out again? I feel like we talked about it, but I can't remember

Reply:
This is just so it reads smoothly. I have to put the name of the project in the middle of the alert but didn't want to hard-code any of the dialogue in english

Reply:
@elizabethhalper  - We can use string interpolation inside the localized strings.  See `projectSettings.messages.saveSuccess` for an example.

Reply:
done!